net/DescriptionViewController.m

```diff
@@ -86,104 +174,527 @@ -(void)issueImageDidLoad
     {
         NSLog(@"in description issue did load: ![self.issueImageView.image isEqual:cItems.issueImage], setOutlet");
         dispatch_async(dispatch_get_main_queue(), ^{
-           self.issueImageView.image = [CurrentItems sharedItems].issueImage; 
+            self.issueImageView.image = [CurrentItems sharedItems].issueImage;
         });
         
     }
 }
 
-
-
--(void)viewDidLoad
+-(void)calculateContentViewStaticHeight
 {
-    CurrentItems *cItems = [CurrentItems sharedItems];
-    [cItems.issueImageDelegates addObject:self];
-    self.navigationController.navigationBar.tintColor = [UIColor whiteColor];
-    self.navigationController.navigationBar.barTintColor = [UIColor bawlRedColor];
-    self.navigationController.navigationBar.titleTextAttributes = @{NSForegroundColorAttributeName : [UIColor whiteColor]};
-    self.navigationController.navigationBar.barStyle = UIBarStyleBlack;
+    self.contentStaticHeight = 0;
+    for (UIView *view in self.viewsVertical)
+    {
+        self.contentStaticHeight += view.frame.size.height;
+        NSLog(@"append view height: %f",view.frame.size.height);
+    }
+    for (NSLayoutConstraint *con in self.constraintsVertical)
+    {
+        self.contentStaticHeight += con.constant;
+        NSLog(@"append consthaint height: %f",con.constant);
+        
+    }
+
 }
 
+
 -(void)setDataToView
 {
-    self.titleLabel.text = [CurrentItems sharedItems].issue.name;
+    Issue *currentIssue = [CurrentItems sharedItems].issue;
+    self.titleLabel.text = currentIssue.name;
     self.titleLabel.textColor = [UIColor bawlRedColor];
```

Comment:
sorry, this is not about user, it is about selected issue. On description title text depends on selected issue, so it is dynamic too.